Add of Calendar, Hour Tables, Multi Virtual Thermostats(Thermostat), Multi Virtual Lights(Eclairage) that use Calendar and Hour Tables

dmayard opened the pull request:
Add PlugIn C# Programs of Calendar, Hour Tables, Multi Virtual Thermostats(Thermostat), Multi Virtual Lights(Eclairage) that use Calendar and Hour Tables

genielabs commented:
Hi @dmayard,
there are some conflicts, please update (probably you'll have to work on programs.xml and factory.zip again). I'll wait merging with this before pushing update to hg update manager.

dmayard commented:
The conflict comes I think because you have added the program 67 in the same time I have added mines. Do you think so ?
My programs.xml has some lines mixes with tjis last prog, I saw when I added the last before prog before mines was 'Meter Watt Poll'
I think Git has some bug ?
Common.csproj has same error. I cannot open with MonoDevellop somes  lines '<<<<<HEAD and >>>>> xxxxxxx' appear. And if I make git pull it says me Uptodate. I cannot get your last files.
Now Common.csproj is ok. but in programs.xml I have a mixing not easy to repair in the last two progs. I think delete the five last prog and retry.Wait for my corrections.
Cheers
Dani

dmayard commented:
Hi Gene,
I think I solve the conflicts issues. Can you merge ?
Thank's
Dani
